ooniprobe/View/TestResults/Details/TestDetailsViewController.m

```diff
@@ -30,6 +31,21 @@ - (void)willMoveToParentViewController:(UIViewController *)parent {
     }
 }
 
+//TODO-UPLOAD connect to constraints (after merge branch)
```

Comment:
What do you mean here with `after merge branch`?

Reply:
This branch is not gonna go in master, is gonna go in another branch with the UX for uploading, that is not yet finished.
(I hope this answer your previous question too)